script.js

```diff
@@ -26,4 +26,93 @@ scrollToLinks.forEach((link) => {
 });
 unionButton.addEventListener('click', showMobileMenu);
 
-crossButton.addEventListener('click', hideMobileMenu);
\ No newline at end of file
+crossButton.addEventListener('click', hideMobileMenu);
+
+document.addEventListener('DOMContentLoaded', () => {
+  const form = document.getElementById('form');
+  const username = document.getElementById('name');
+  const email = document.getElementById('email');
+
+  function setError(elm, msg) {
+    const inputControl = elm.parentElement;
+    const errorDiv = inputControl.querySelector('.error');
+    errorDiv.innerText = msg;
+    inputControl.classList.add('error');
+  }
+
+  function clearError(elm) {
+    const inputControl = elm.parentElement;
+    const errorDiv = inputControl.querySelector('.error');
+    errorDiv.innerText = '';
+    inputControl.classList.remove('error');
+  }
+
+  function isLowerCase(str) {
+    return str === str.toLowerCase();
+  }
+
+  function validateFormInputs() {
+    const usernameValue = username.value.trim();
+    const emailValue = email.value.trim();
+
+    clearError(username);
+    clearError(email);
+
+    if (usernameValue === '') {
+      setError(username, 'Username is required');
+    }
+    if (emailValue === '') {
+      setError(email, 'Email is required');
+    } else if (!isLowerCase(emailValue)) {
+      setError(email, 'Email must be in lowercase');
+    } else {
+      // Validation is OK, submit the form
+      form.submit();
+    }
+  }
+
+  form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    validateFormInputs();
+  });
+});
+
+// document.addEventListener('DOMContentLoaded', () => {
+//   const form = document.getElementById('form');
+//   const username = document.getElementById('name');
+//   const email = document.getElementById('email');
+
+//   function setError(elm, msg) {
+//     const inputControl = elm.parentElement;
+//     const errorDiv = inputControl.querySelector('.error');
+//     errorDiv.innerText = msg;
+//     inputControl.classList.add('error');
+//   }
+
+//   function clearError(elm) {
+//     const inputControl = elm.parentElement;
+//     const errorDiv = inputControl.querySelector('.error');
+//     errorDiv.innerText = '';
+//     inputControl.classList.remove('error');
+//   }
+
+//   function validateFormInputs() {
```

Comment:
Great job implementing the form validation but please make sure to remove all commented out code to keep your codebase clean👍